Photon-Tinker/WelcomeViewController.swift

```diff
@@ -132,6 +132,7 @@ class WelcomeViewController: UIViewController, ParticleSetupMainControllerDelega
             if let vc = ParticleSetupMainController(authenticationOnly: true)
             {
                 vc.delegate = self
+                vc.startWithLogin = true
```

Comment:
Making the login screen the default -- was that a design discussion I missed, or did you decide that on your own?

Reply:
Good catch. Actually this was something that Mirande reported and I that was mentioned few times during the stand-ups but without much debating I went ahead and implemented that as it totally makes sense.